nifi-nar-bundles/nifi-mqtt-bundle/nifi-mqtt-processors/src/test/java/org/apache/nifi/processors/mqtt/common/MqttTestClient.java

```diff
@@ -195,4 +218,296 @@ public String getServerURI() {
     public void close() throws MqttException {
 
     }
+
```

Comment:
There is a lot of boilerplate looking javadocs.  Are these copied/pasted from somewhere?  Where is that?

Reply:
Hey @joewitt! The newer version adds methods to the IMqttClient interface and I auto-imported them. It pulled in the JavaDocs as well (could be the IDEA default?) and I originally left them as I was sure if they were required. I have removed them in the new update as it seems unnecessary to include them for the MqttTestClient. Would you agree?